ruff.toml

```diff
@@ -97,4 +99,12 @@ section-order = [
 
 
 [format]
-line-ending = "lf"
\ No newline at end of file
+line-ending = "lf"
+
+[lint.flake8-tidy-imports]
+banned-module-level-imports = [
+    "logging",
+]
```

Comment:
This seems to give duplicate errors:

```
$ ruff check src
src/objects/utils/middleware.py:1:8: TID253 `logging` is banned at the module level
  |
1 | import logging
  |        ^^^^^^^ TID253
2 | import os
  |

src/objects/utils/middleware.py:1:8: TID251 `logging` is banned: Use `structlog.stdlib.get_logger(__name__)` instead of the stdlib logging module.
  |
1 | import logging
  |        ^^^^^^^ TID251
2 | import os
```

I think just `lint.flake8-tidy-imports.banned-api` is sufficient

```suggestion
```